meta-mender-core/recipes-devtools/dosfstools/dosfstools_4.2.bb

```diff
@@ -0,0 +1,32 @@
+# dosfstools OE build file
+# Copyright (C) 2004-2006, Advanced Micro Devices, Inc.  All Rights Reserved
+# Copyright (C) 2015, SÃ¶ren Brinkmann <soeren.brinkmann@gmail>  All Rights Reserved
```

Comment:
Looks like a typo to me

Reply:
Hmm, I wasn't sure what to do about it myself, but it is like this in the upstream poky recipe for `4.1`

Reply:
okey, then